app/views/shared/vacancy/_jobseeker_view.html.haml

```diff
@@ -15,6 +15,9 @@
 
   .govuk-grid-column-two-thirds
 
+    - if @vacancy.organisations.many?
+      = render(Jobseekers::NotificationComponent.new(content: t('jobs.notifications.content.multi_school_job_notification_html')))
```

Comment:
I see Cesi has recommended using a partial instead, but it's worth knowing that a ViewComponent can be given a method `render?` which then lets you hide the decision whether or not to render, resulting in a cleaner, less logic-ful template. We've used it elsewhere in the app if you want to look at some examples.

@cesidio This might mean that there is in fact some benefit to using a ViewComponent here, because we remove 1 line of logic from the template. What do you think?

Reply:
Now I remember @cpjmcquillan mentioned there was some complexity introduced by nesting components inside each other, but I don't know what complexity that is... care to comment?